openpathsampling/snapshot.py

```diff
@@ -117,306 +91,73 @@ def copy(self):
         content (e.g. Configuration object) will not.
 
         """
-        this = AbstractSnapshot(is_reversed=self.is_reversed, topology=self.topology)
+        this = self.__class__.__new__(self.__class__)
+        AbstractSnapshot.__init__(this, topology=self.topology)
         return this
 
-    def reversed_copy(self):
-        """
-        Returns a shallow reversed copy of the instance itself. The
-        contained configuration and momenta are not copied and the momenta
-        are marked reversed.
-
-        Since Snapshots are made in pairs this will also lead to a new
-        (non-)reversed copy!
-
-        Returns
-        -------
-        Snapshot()
-            the reversed shallow copy
-        """
+    def create_reversed(self):
+        this = self.copy()
+        this._reversed = self
+        return this
 
-        obj = self.copy()
-        return obj.reversed
 
+class FeatureSnapshot(AbstractSnapshot):
+    def copy(self):
+        return super(FeatureSnapshot, self).copy()
 
-# =============================================================================
-# SIMULATION SNAPSHOT (COMPLETE FRAME WITH COORDINATES AND VELOCITIES)
-# =============================================================================
 
-@lazy_loading_attributes('configuration', 'momentum', '_reversed')
-class Snapshot(AbstractSnapshot):
+def snapshot_factory(name, features, description=None):
```

Comment:
My gut wants the factory to be named `SnapshotFactory`. Not sure why, since I feel like this naming fits the letter of the law better. Still, it feels weird. Thoughts?